datashuttle/datashuttle.py

```diff
@@ -281,11 +283,13 @@ def make_config_file(
         settings will be used each time the datashuttle is opened.
 
         :param local_path:                  path to project dir on local machine
-        :param remote_path:                 path to project directory on remote machine. Note this cannot
-                                            include ~ home directory syntax, must contain the full path (
-                                            e.g. /nfs/nhome/live/jziminski)
+        :param remote_path_local:           Full filepath to local filesystem (e.g. mounted drive) dir
```

Comment:
Is there any way of ssh'ing to the remote machine to expand the full path, so we could support the `~` syntax?

Reply:
Not critical, happy to enforce the use of full paths.

Reply:
I think it would be nice and if I recall correctly was based on limitation of pyftpsync but rclone (99% sure) does support ~ syntax, will look into before merging into master #10